lms/security.py

```diff
@@ -177,7 +177,9 @@ def identity(self, request):
         if lti_user is None:
             return Identity("", [])
 
-        permissions = [Permissions.LTI_LAUNCH_ASSIGNMENT, Permissions.API]
+        permissions = []
+        if lti_user.is_learner or lti_user.is_instructor or lti_user.is_admin:
```

Comment:
Maybe we could define a new property that wraps and internally calls these three, so that we can do `if lti_user.has_valid_role`, or something in those lines.

Just something to consider in future.

Reply:
Yes :+1: I thought of that but was having trouble coming with a good name.

Reply:
Names are hard 😅